UI/Pages/Game.razor

```diff
@@ -19,23 +20,30 @@
   }
   else
   {
-    <div class="flex justify-center gap-9 pb-6">
-      @if (_host is not null)
-      {
-        <PlayerCard Player="@_host" ClientConnectionId="@_clientConnectionId"/>
-      }
-
-      @if (_guest is not null)
-      {
-        <PlayerCard Player="@_guest" ClientConnectionId="@_clientConnectionId"/>
-      }
-    </div>
-
-    <GameGrid
-      GridCells="@_gameBoard"
-      ActiveMark="@_activeMark"
-      OnClickFunction="@MakeMove"
-      CanMakeMove="@CanMakeMove()"/>
+    if (_gameCanStart)
+    {
+      <div class="flex justify-center gap-9 pb-6">
+        @if (_host is not null)
```

Comment:
Remove the if statements that check host and guest since _gameCanStart is false if any is null